[build issue] build on macOS fails perhaps looking for octave

On a fresh macOS 15.5 machine.
./build_osx.sh stops with Error 2.
To see what's going on I commented out the three -j$(sysctl -n hw.logicalcpu) so it builds in a single thread.

It fails with: 
```
checking for getpagesize... yes
checking for setlocale... yes
checking for pipe... yes
checking for waitpid... yes
checking for floor in -lm... yes
checking for floor... yes
checking for ceil... yes
checking for fmod... yes
checking for lround... yes
checking for lrint... yes
checking for lrintf... yes
checking for octave... no
checking for mkoctfile... no
checking for octave-config... no
./configure: line 21896: PKG_PROG_PKG_CONFIG: command not found
./configure: line 21911: syntax error near unexpected token `FLAC_CFLAGS,'
./configure: line 21911: `_PKG_CONFIG(FLAC_CFLAGS, cflags, flac >= 1.3.1)'
make[2]: *** [sndfile-prefix/src/sndfile-stamp/sndfile-configure] Error 2
make[1]: *** [CMakeFiles/sndfile.dir/all] Error 2
make: *** [all] Error 2
```
We discussed this in the past and the feeling was that octave should not be required.

--- a/README.md
+++ b/README.md
@@ -149,15 +149,24 @@ one to build FreeDV for ARM as well as for Intel Windows systems.
 Using MacPorts, most of the appropriate dependencies can be installed by:
 
 ```
-$ sudo port install automake git libtool sox +universal cmake wget
+$ sudo port install automake git libtool sox +universal cmake wget pkgconf
 ```
 
 and on Homebrew:
 
 ```
-$ brew install automake libtool git sox cmake wget
+$ brew install automake libtool git sox cmake wget pkgconf
 ```
 
 Once the dependencies are installed, you can then run the `build_osx.sh` script inside the source tree to build
 FreeDV and associated libraries (codec2, hamlib). A FreeDV.app app bundle will be created inside the build_osx/src
 folder which can be copied to your system's Applications folder.
+
+*Note: for distribution, code signing is required. The following commands can be run to enable this:*
+
+```
+CODESIGN_IDENTITY=[identity in your keychain] UNIV_BUILD=1 ./build_osx.sh
+cd build_osx
+make release
+```
+

--- a/USER_MANUAL.md
+++ b/USER_MANUAL.md
@@ -857,6 +857,8 @@ LDPC | Low Density Parity Check Codes - a family of powerful FEC codes
 3. Build system:
     * Update Hamlib to v4.6.4. (PR #989)
     * macOS: Upgrade PyTorch to 2.7 (and NumPy to 2.3). (PR #1003)
+4. Documentation:
+    * Add pkgconf dependency for macOS instructions. (PR #1013)
 
 ## V2.0.1 July 2025
 

